src/main.rs

```diff
@@ -163,12 +166,18 @@ fn main() {
     }
 }
 
-fn control_time(days: &Arc<Mutex<i32>>, stats: &Arc<Mutex<Stats>>) {
+fn control_time(days: &Arc<Mutex<i32>>, stats: &Arc<Mutex<Stats>>, fire: &Arc<Mutex<Fire>>) {
     let now = Instant::now();
     let days = Arc::clone(&days);
     let stats = Arc::clone(&stats);
+    let fire = Arc::clone(&fire);
+
     thread::spawn(move || loop {
         thread::sleep(Duration::from_secs(10));
+
+        let mut fire_lock = fire.lock().unwrap();
+        fire_lock.pass_time();
```

Comment:
What do you think about moving the `10` seconds to a variable and passing it to `pass_time`? Then you can remove the hardcoded `10` inside that method 😄

Reply:
I guess, but at the moment I don't see any case where it would be a different time, so I'd just be hardcoding the `10` somewhere else 🤷‍♀